assets/pages/index.vue

```diff
@@ -1,13 +1,9 @@
 <template>
   <div>
-    <section class="hero is-small mt-4">
-      <div class="hero-body">
-        <div class="container">
-          <div class="columns">
-            <div class="column is-narrow" v-if="secured">
-              <a class="button is-primary is-small" :href="`${base}/logout`">{{ $t("button.logout") }}</a>
-            </div>
-          </div>
+    <section class="is-small mt-1" v-if="secured && isMobile">
```

Comment:
You said you will come back to this, right? We can do something similar in mobile and have a row for all the buttons.

Reply:
You can leave the button showing only in the mobile version. But I already have a revision on the buttons for the mobile menu.